postman_tests/data/elastic_search/city.csv

```diff
@@ -0,0 +1,1206 @@
+value,variants
```

Comment:
If possible, let's limit this to top 100 examples or so. Otherwise, it unnecessarily increases repo size

Reply:
I think all entities during the test should be prefixed with `ptest_` to avoid any conflicts with the current running environment.

In an ideal scenario, the `run_tests` should bring up a separate set of docker containers just for testing and destroy them once tests are run. But for now, if we are using the already up containers, prefixing is necessary to reduce the chances that we don't end up deleting already existing data

Reply:
Limiting to 100 records now.